vm_automation/esxiVm.py

```diff
@@ -8,6 +8,8 @@
 import random
 import atexit
 import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)

```

Comment:
Ick, is it standard practice in Python to put things like this in the `imports` section of code? :-(

May just be my preference but would it make sense to treat configuration items like this like their own section below `import` and still in global space?